web-ui/etc/config.yaml

```diff
@@ -4,10 +4,8 @@ verbosity:  1
 web_ui: # {
   #port: 3000
   port: 5173
-# } web_ui
 
-# Access to web-services (REST and JsonRPC)
-web_services: # {
-  host: api.ayia.nibious.com
-  port: 443
-#} web_services
+  api_map: #{
+    versions: http://localhost:4000/api/v2/versions
```

Comment:
Should this be put back to `api.ayia.nibious.com`?

Reply:
Yes. You are correct.
I'll make the change.